### ISSUE TITLE
command line "-o" option bug.

### ISSUE THREAD

**Issue by oaspireo** [indexzero/http-server#264]
My English is not good, so I will describe this issue with code.

`$ node node_modules/http-server/bin/http-server -o`
when I use "-o" option, browser will open **http:////127.0.0.1:8080**

in **http-server/bin/http-server**

```
    if (argv.o) {
      console.log(protocol,"-",canonicalHost,"-",port);
      opener(
        protocol + '//' + canonicalHost + ':' + port,
        { command: argv.o !== true ? argv.o : null }
      );
    }
```

and console appears 

```
Starting up http-server, serving ./
Available on:
  http://10.1.1.32:8080
  http://127.0.0.1:8080
Hit CTRL-C to stop the server
http:// - 127.0.0.1 - 8080
```

**protocol** already has "//" in it.

**Comment by LenHannam** [comment]
I made my default browser Google Chrome and the problem went away. So, I suspect Internet Explorer is the issue (I'm running Windows 7 with I.E. 11)

**Comment by thornjad** [comment]
Fixed in #374
